keras_hub/src/layers/modeling/non_max_supression.py

```diff
@@ -519,22 +531,16 @@ def mask_invalid_detections(bounding_boxes):
         returned value will also return `tf.RaggedTensor` representations.
     """
     # ensure we are complying with Keras bounding box format.
-    info = validate_format.validate_format(bounding_boxes)
-    if info["ragged"]:
-        raise ValueError(
-            "`bounding_box.mask_invalid_detections()` requires inputs to be "
-            "Dense tensors. Please call "
-            "`bounding_box.to_dense(bounding_boxes)` before passing your boxes "
-            "to `bounding_box.mask_invalid_detections()`."
-        )
+    validation.validate_bounding_boxes(bounding_boxes)
```

Comment:
Should we consider making this public API in Keras? (Though we can keep this usage for now)

Reply:
Or could we trigger this with public API? E.g. `keras.utils.bounding_boxes.convert_format` to `self.bounding_box_format` like we do in call?

Reply:
I think need to either remove this or duplicate it here. `import keras.src.layers.preprocessing.image_preprocessing.bounding_boxes` will means KerasHub will not even import for older than the latest version of keras (including the keras currently preinstalled in colab images).

We don't want that.

Reply:
Done